dev-haskell/cabal-install/cabal-install-3.2.0.0.ebuild

```diff
@@ -46,16 +46,11 @@ DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-1.22.2.0
 "
 
-PATCHES=( "${FILESDIR}"/${P}-rankntypes.patch )
```

Comment:
This patch should not be dropped. It is required for the package to successfully build on ghc-8.10.

Reply:
@solpeth I think anything I did regarding `3.2.0.0` was by accident, reverting all changes.